lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,47 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server:                 # https://webpack.js.org/configuration/dev-server/
+    host: localhost           # Dev server host
+    port: 8080
+    https: false
+
+  paths:                       # ~ = Rails.root
+    source: app/javascript     # ~/:source
+    entry: packs               # ~/:source/:entry
+    output: packs              # ~/public/:output
+    manifest: manifest.json    # ~/public/:output/:manifest
+    config: config/webpack     # ~/:config
```

Comment:
Yepp, that would work 👍  Although a bit repetitive and verbose if we start to have many options, especially for the dev server -`dev_server-host`, `dev_server_port`, `dev_server_hot`, `dev_server_https`. Namespace gives a bit flexibility on the JS side too - it's easy to extract objects using - `{ dev_server, paths }`

```yml
default: &default
  dev_server_host: localhost           
  dev_server_port: 8080
  dev_server_https: false
  dev_server_hot: false
                
  source_path: app/javascript  
  source_entry_path: packs        
  public_output_path: packs     
```

VS

```yml
default: &default
  dev_server:                 # https://webpack.js.org/configuration/dev-server/
    host: localhost           # Dev server host
    port: 8080
    https: false
    hot: false

  paths:                       # ~ = Rails.root
    source: app/javascript     # ~/:source
    entry: packs               # ~/:source/:entry
    output: packs              # ~/public/:output
```